velox/docs/spark_functions.rst

```diff
@@ -0,0 +1,73 @@
+***********************
+Spark Functions
+***********************
+
+.. toctree::
+    :maxdepth: 2
+
+    functions/spark/math
+    functions/spark/bitwise
+    functions/spark/comparison
+    functions/spark/string
+    functions/spark/datetime
+    functions/spark/array
+    functions/spark/map
+    functions/spark/regexp
+    functions/spark/binary
+    functions/spark/json
+    functions/spark/aggregate
+
+Here is a list of all scalar and aggregate Spark functions available in Velox.
+Function names link to function descriptions. Check out coverage maps
```

Comment:
> Function names link to function descriptions.

Does that work? What about function names that are the same between Presto and Spark?

Reply:
I use js namespace to describe Spark function which has the same name with Presto temporarily.